ui/src/App.css

```diff
@@ -15,6 +15,20 @@ body,
 }
 
 .logo {
-  margin-left: -15px;
-  margin-top: 5px;
+  width: 37px;
```

Comment:
rem is used in most of the components, let's use a common unit.

Reply:
Updated all px to rem.